src/iris/bin/run_server.py

```diff
@@ -8,7 +8,7 @@
 import sys
 import multiprocessing
 import gunicorn.app.base
-from gunicorn.six import iteritems
+from six import iteritems
```

Comment:
The py3 branch already breaks python 2 compatibility. We can use items() instead and avoid importing the six library.